packages/react-components/src/search/EventSearch/EventSearchPageLayout.js

```diff
@@ -40,6 +41,7 @@ const Layout = ({
         <DataHeader availableCatalogues={config.availableCatalogues} style={{ borderBottom: '1px solid #ddd' }} />
         <div css={cssFilter({ theme })}>
           <FilterBar config={config}></FilterBar>
+          <ClearFilter/>
```

Comment:
thanks @qifeng-bai 

Can we remove the "Reset" button from the PR ?
I think this button needs a re-think and some input from @MortenHofft 
It would be nice to come up with a common design element for events / occurrences / other  search filters.

Reply:
Done